src/dataflow/operators/ros/from_ros_operator.rs

```diff
@@ -0,0 +1,70 @@
+use crate::dataflow::{
+    stream::{WriteStream, WriteStreamT}, operator::Source,
+    Data, Message 
+};
+use serde::Deserialize;
+use std::sync::{Arc, Mutex};
+
+/// Subscribes to a ROS topic and outputs incoming messages to an ERDOS stream using the
+/// provided message conversion function.
+/// 
+/// Conversion function should convert a ROS type which implements the [`rosrust::Message`] 
+/// trait and return a ERDOS [`Message`] containing data of a Rust data type. See [`rosrust_msg`]
+/// for a variety of supported standard ROS messages.
+/// 
+/// # Example
+/// The following example shows a conversion function which takes a [`rosrust_msg::sensor_msgs::Image`]
+/// and returns an ERDOS message containing [`Vec<u8>`] a vector of bytes.
+/// 
+/// ```
+/// fn ros_to_erdos(input: &rosrust_msg::sensor_msgs::Image) -> Message<Vec<u8>> {
+///     Message::new_message(Timestamp::Time(vec![0 as u64]), input.data)
+/// }
```

Comment:
Could you also include an example that initializes a `FromRosOperator` with the function?